modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Link.js

```diff
@@ -1,10 +1,13 @@
 import Link from '@tiptap/extension-link';
 export default (options) => {
   return Link.extend({
-    defaultOptions: {
-      openOnClick: false,
-      linkOnPaste: true,
-      HTMLAttributes: {}
+    addOptions() {
+      return {
+        ...this.parent?.(),
+        openOnClick: false,
+        linkOnPaste: true,
+        HTMLAttributes: {}
+      };
```

Comment:
now that we use the expected `addOptions` way, `openOnClick` is now correctly taken into account